docs/src/components/AppBar.tsx

```diff
@@ -45,7 +45,9 @@ export function AppBar() {
         <IconLinkButton useNextLink href="/" label="Pigment CSS" size={3}>
           <PigmentIcon />
         </IconLinkButton>
-        <DocsVersionSelector currentVersion={currentVersion} versions={supportedVersions} />
+        {supportedVersions.length > 1 && (
```

Comment:
Instead of hiding we can have a span to show the current version.

Reply:
Will update in a followup.